Satellite-im/UplinkWeb
# chore(build): fix warnings

### What this PR does 📖

Fix this warning on prod build

<img width="1512" alt="Captura de ecrã 2024-08-30, às 13 21 03" src="https://github.com/user-attachments/assets/8791054c-2be5-4b97-91c6-6bfa0bce3a05">


How to test
- `npm run build`
- above warning should not appear

before

<img width="1512" alt="Captura de ecrã 2024-08-30, às 13 21 03" src="https://github.com/user-attachments/assets/8791054c-2be5-4b97-91c6-6bfa0bce3a05">


after

<img width="1385" alt="Captura de ecrã 2024-08-30, às 13 22 58" src="https://github.com/user-attachments/assets/62e6421d-bb07-4548-af3c-1b59d2b4a65b">


everything should be the same as dev
